packages/application-shell/package.json

```diff
@@ -51,9 +51,9 @@
     "@commercetools-frontend/sentry": "7.0.0",
     "@commercetools-frontend/storage": "7.0.0",
     "@commercetools-frontend/url-utils": "7.0.0",
-    "@flopflip/launchdarkly-adapter": "2.4.5",
-    "@flopflip/memory-adapter": "1.1.0",
-    "@flopflip/react-broadcast": "7.2.4",
+    "@flopflip/launchdarkly-adapter": "2.4.6",
+    "@flopflip/memory-adapter": "1.1.1",
+    "@flopflip/react-broadcast": "7.2.5",
```

Comment:
Mostly internal dep updates and some simplifications in how we handle adapter state. Nothing which affects the apps.